Tasks/Common/azure-blobstorage-artifactProvider/azureBlobStorageProvider.ts

```diff
@@ -2,9 +2,13 @@ import path = require('path');
 import azureStorage = require('azure-storage');
 import fs = require('fs');
 import models = require('artifact-engine/Models');
+import store = require('artifact-engine/Store');
 import tl = require('vsts-task-lib/task');
 
 export class AzureBlobProvider implements models.IArtifactProvider {
+
+    public artifactItemStore: store.ArtifactItemStore;
```

Comment:
where is this used?

Reply:
not used currently, should be used to set downloadfilesize.